reactor-client/src/main/java/com/aerospike/client/reactor/IAerospikeReactorClient.java

```diff
@@ -0,0 +1,556 @@
+package com.aerospike.client.reactor;
+
+import com.aerospike.client.*;
+import com.aerospike.client.policy.*;
+import com.aerospike.client.query.KeyRecord;
+import com.aerospike.client.query.Statement;
+import com.aerospike.client.reactor.dto.KeyExists;
+import com.aerospike.client.reactor.dto.KeyObject;
+import com.aerospike.client.reactor.dto.KeysExists;
+import com.aerospike.client.reactor.dto.KeysRecords;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.List;
+
+
+public interface IAerospikeReactorClient {
+
+	void close();
```

Comment:
maybe we could extend `Closeable` interface?

Reply:
fixed